Dockerfile.dist

```diff
@@ -41,7 +41,7 @@ RUN go mod download
 # Now copy everything including .git
 COPY . .
 
-RUN /build/build.sh "${BUILD_LIST}"
+RUN /build/build.sh "${BUILD_LIST}" redhat
```

Comment:
Would it make sense to make this `"redhat"` an ARG?

Reply:
Maybe not needed right now, but I like it. Added to new revision.